frontend/packages/dev-console/src/components/import/pipeline/PipelineSection.tsx

```diff
@@ -15,7 +16,14 @@ type PipelineSectionProps = {
 const PipelineSection: React.FC<PipelineSectionProps> = ({ flags }) => {
   const { values } = useFormikContext<FormikValues>();
 
-  if (flags[FLAG_OPENSHIFT_PIPELINE]) {
+  const canListPiplines = useAccessReview({
+    group: PipelineModel.apiGroup,
+    resource: PipelineModel.plural,
+    namespace: 'openshift',
```

Comment:
I think we also need to stop doing this for our own sake. If the powers that be decide `openshift` is getting too crowded and it should be in something like `openshift-pipelines`, we need to be able to not start a scavanging hunt for the use of `'openshift'`. Please create a constant in `dev-console/src/const.ts` and use it here as well as update the reference [here](https://github.com/openshift/console/blob/master/frontend/packages/dev-console/src/components/import/pipeline/PipelineTemplate.tsx#L35).

Suggested variable name (feel free to use something else if you can think of a better name):
```
export const CLUSTER_PIPELINE_NS = 'openshift';
```

Reply:
@andrewballantyne Added ```openshift``` in ```CLUSTER_PIPELINE_NS``` const.